config.js

```diff
@@ -67,25 +85,42 @@ exports.updateFromArgv = function() {
         // --rejectnew --rejectnewmsg "this bot is not playing today because blablablah, try again at x time, sorry"
         .describe('rejectnewfile', 'Reject new challenges if file exists (checked each time, can use for load-balancing)')
         .describe('boardsize', 'Board size(s) to accept')
+        .describe('boardsize', 'Board size(s) to accept')
```

Comment:
It seems lines 87 and 88 are duplicates of the same line here.

Reply:
indeed :+1:

Reply:
thank you, fixed :+1: